doc/en/components/charts/chart-features-highlighting.md

```diff
@@ -7,7 +7,58 @@ mentionedTypes: []
 
 # $PlatformShort$ Chart Highlighting
 
-<!-- TODO combine
-category-chart-highlighting.md
-data-chart-series-highlighting.md
--->
\ No newline at end of file
+<code-view style="height: 500px" 
+           data-demos-base-url="{environment:dvDemosBaseUrl}" 
+           iframe-src="{environment:dvDemosBaseUrl}/charts/category-chart-column-chart-with-highlighting" 
+           alt="$PlatformShort$ Highlighting Example" 
+           github-src="charts/category-chart/category-chart-column-chart-with-highlighting">
+</code-view>
+
+<div class="divider--half"></div>
+
+
+## Highlight Layers
+
+The $ProductName$ category chart component can enable three types of highlighting when hovering over items.
+
+1. Series Highlighting will highlight the single data point when the pointer is positioned over it.
```

Comment:
change "the single data point"
to "the single data point represented by a marker or column"

Reply:
pending